.ci/openshift-ci/test-azure-no-pat-oauth-flow-raw-devfile-url.sh

```diff
@@ -16,6 +16,8 @@ set -ex
 # only exit with zero if all commands of the pipeline exit successfully
 set -o pipefail
 
+echo -e "\e[1;3;4;32m[INFO] OpenShift CI infrastructure is ready.\nTest is run.\e[0m"
```

Comment:
`Test is run` -> `Running test`?

Reply:
Yeah.. 
Maybe `Test is running` is more preferable, you mean this ?

Reply:
Not a big deal but, `Test is running` sounds like you request a current status and the response is: the test is running. However, this log line is more like a next step indicator, so I think `Running test(s)` sounds more suitable.

Reply:
OK, got it